### ISSUE TITLE
On aarch64 architecture. 'tokio-runtime-worker' panicked. The feature 'websocket' is not compiled in this binary. 

### ISSUE THREAD

**Issue by tlze** [rapiz1/rathole#308]
rathole v0.5.0

**Describe the bug**

Run: RUST_BACKTRACE=1 rathole /etc/rathole/config.toml 
thread 'tokio-runtime-worker' panicked at 'The feature 'websocket' is not compiled in this binary. Please re-compile rathole'

**To Reproduce**
1. RUST_BACKTRACE=1 rathole /etc/rathole/config.toml
 
**Configuration**
Configuration used to reproduce the behavior:
[client]
remote_addr = "1.2.3.4:5000"
default_token = "**456123**"

[client.transport]
type = "websocket"
[client.transport.websocket]
tls = false

[client.services.pwg]
type = "udp"
local_addr = "127.0.0.1:1838"

**Logs**

thread 'tokio-runtime-worker' panicked at 'The feature 'websocket' is not compiled in this binary. Please re-compile rathole', src/helper.rs:39:5
stack backtrace:
2023-11-16T03:25:59.518746Z  INFO config_watcher{path="/etc/rathole/config.toml"}: rathole::config_watcher: Start watching the config
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
Aborted

**Environment:**
 - OS: debian 12
- `rathole --version` output: rathole v0.5.0
- CPU architecture: aarch64

**Comment by fernvenue** [comment]
Read [the release note](https://github.com/rapiz1/rathole/releases/tag/v0.5.0) carefully:

> Because of the maintenance burden of linking TLS libraries, we don't compile the TLS feature in the release for embedded platforms anymore. If you need to use it, please build from the source by yourself. We will add it back when a pure rust tls library or a good cross-platform TLS libraries linking solution is available.

And as you can see:

https://github.com/rapiz1/rathole/blob/65b27f076cfb9789873223b803c6c394f91fd47d/Cargo.toml#L24-L25

TLS is required by websocket feature, so that's expected behavior.

TLDR, compile binary by yourself.

**Comment by tlze** [comment]
Thanks, I got it.
